### DISCUSSION TITLE
Windows 32/64 bit builds

### DISCUSSION THREAD

**Discussion by hbbernardo** [fraunhoferhhi/vvdec#38] [Q&A]
Are there any Windows 32/64 bit builds around?

I am new to this.

Please, let me know.

Thanks

**Comment by berbunda** [comment]
@hbbernardo 
Hi! I downloaded **CMake** and **MS VS 16 2019**. Made some editions in **Makefile** and **CMakeLists.txt**. Run **CMake**. It made some **vcxproj** files. I run **ALL_BUILD.vcxproj** and built exe-files of vvdec and vvenc. But I cannot to check workability of files because encoded files is not playable with players(yes, i am naive and stupid). I am not sure do it possible to upload files to project vvdec and vvenc. 
_For understanding: I have never have experience with coding, programming etc_
